Makefile

```diff
@@ -24,9 +25,9 @@ functions-build:
 check-headers-file:
 	scripts/check-headers-file.sh
 
-production-build: build check-headers-file ## Build the production site and ensure that noindex headers aren't added
+production-build: accessibility-check build check-headers-file ## Build the production site and ensure that noindex headers aren't added
```

Comment:
What is accessibility-check vs. accessibility-page?

Reply:
This is just a WIP at the moment. I'll let you know when it's ready for review!

Reply:
👍

Reply:
Love the enthusiasm though 😄